pmm/pmm2-ovf-test.groovy

```diff
@@ -1,10 +1,11 @@
-void runUITests(CLIENT_VERSION, CLIENT_INSTANCE, SERVER_IP, GIT_BRANCH) {
-    stagingJob = build job: 'pmm2-ui-tests', parameters: [
+void runUITests(CLIENT_VERSION, CLIENT_INSTANCE, SERVER_IP, GIT_BRANCH, CLIENTS) {
+    stagingJob = build job: 'pmm2-ui-tests-temp', parameters: [
```

Comment:
remember to change it back to normal and disable the temp job.

Reply:
done